package.json

```diff
@@ -75,7 +75,7 @@
   },
   "dependencies": {
     "camel-case": "^4.1.2",
-    "clean-css": "5.2.0",
+    "clean-css": "~5.3.2",
```

Comment:
You really should have used `^` here so that when 5.4.x etc is out no one needs to change the package here.

Reply:
Well, if they offer to update the deps in a timely manner, sure. Otherwise, better use `^` and when tests break, someone updates the fixtures. That was the consumers won't have to wait for a new html-minifier-terser release.

Both solutions are a compromise, of course; it depends from which point of view one sees the issue. :)

Reply:
Yes, I didn’t want to use `~` or `^` in case of clean-css, just pin to a fixed version. since they introduced a breaking change in semver minor. But if it a one time thing like an accident or a bug fix actually. May be we can ignore this once.

As this might allow less maintenance releases from our end, also allows users to stay upto date on security patches.

I am fine on both options have a caret or pin it. But if clean-css doesn’t follow semver or this happened more than once, I recommend pin it.

Reply:
Yeah, semver is hard to follow, I've been there myself 😛

Thanks for considering!